lms/djangoapps/instructor_task/tasks_helper/misc.py

```diff
@@ -151,7 +152,10 @@ def cohort_students_and_upload(_xmodule_instance_args, _entry_id, course_id, tas
     # Iterate through rows to get total assignments for task progress
     with DefaultStorage().open(task_input['file_name']) as f:
         total_assignments = 0
-        for _line in unicodecsv.DictReader(UniversalNewlineIterator(f)):
+        csv_content = f.read().decode('utf-8') if six.PY3 else UniversalNewlineIterator(f)
+        reader = (csv.DictReader(csv_content.splitlines()) if six.PY3
+                  else unicodecsv.DictReader(csv_content))
```

Comment:
I realize this is a pre-existing problem, but is there a good reason not to use the `encoding='utf-8'` option passed into the same function below?